src/components/search-form/components/packages-filter-accordion/packages-filter-accordion.js

```diff
@@ -0,0 +1,83 @@
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Accordion,
+  Icon,
+  FilterAccordionHeader,
+} from '@folio/stripes/components';
+
+import { MultiSelectionFilter } from '@folio/stripes/smart-components';
+
+import FacetOptionFormatter from '../../../facet-option-formatter';
+
+const propTypes = {
+  activeFilters: PropTypes.object.isRequired,
+  dataOptions: PropTypes.array.isRequired,
+  disabled: PropTypes.bool.isRequired,
+  isLoading: PropTypes.bool.isRequired,
+  onUpdate: PropTypes.func.isRequired,
+};
+
+const PackagesFilterAccordion = ({
+  activeFilters,
+  dataOptions,
+  disabled,
+  isLoading,
+  onUpdate,
+}) => {
+  const {
+    packageIds = [],
+  } = activeFilters;
+
+  const packagesList = Array.isArray(packageIds)
+    ? packageIds
+    : packageIds.split(',');
+
+  packagesList.sort();
+
+  const itemToString = option => option?.label || '';
+
+  return (
+    <div
+      role="tab"
+      data-testid="packagesFilter"
+    >
+      <FormattedMessage id="ui-eholdings.packages.filter">
+        {([label]) => (
+          <Accordion
+            label={label}
+            id="packagesFilter"
+            separator={false}
+            closedByDefault
+            header={FilterAccordionHeader}
+            displayClearButton={!!packagesList.length}
+            onClearFilter={() => onUpdate({ packageIds: undefined })}
+          >
+            {isLoading && !disabled
```

Comment:
Instead of showing a spinner when facets are loading - can we disable the filter?
We usually hide elements that can't be accessed at all, like when a permission is missing etc.
But in our situation user can access the filter if they perform a search so I think just disabling it will be a better experience